client/src/components/itemList/ItemListHeader.js

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+import React from 'react';
+
+export const Header = styled.div`
+  width: 100%;
+  height: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  font-size: 1.2rem;
+  border-bottom: 2px solid #a58f73;
+  margin-bottom: 20px;
+`;
```

Comment:
```suggestion
export const Header = styled.div`
  width: 100%;
  height: 50px;
  display: flex;
  align-items: center;
  justify-content: space-around;
  font-size: 1.2rem;
  border-bottom: 2px solid #a58f73;
  margin-bottom: 20px;
  & > p {
  flex-basis: 25%;
  text-align: center;
  }
`;
```

The same idea of having each child element would also be needed here to have every `p` with an equal width.